tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "jsx": "react-jsx",
     "declaration": true,
     "outDir": "./dist",
-    "emitDeclarationOnly": true,
+    "noEmit": true,
```

Comment:
Wondering the reason for this change ? :) does it mean dist will no longer have .d.ts files ?

Reply:
It's reverting to the original setting, where it will not try to extract types out of js files. This will ensure that the complex Proptypes that we have in JS files won't be incorrectly converted.